mmseg/models/losses/tversky_loss.py

```diff
@@ -31,6 +32,8 @@ def tversky_loss(pred,
                 alpha=alpha,
                 beta=beta,
                 smooth=smooth)
+            if gamma > 1.0:
+                tversky_loss **= gamma
```

Comment:
Should it be `1 / gamma`?
At the official implementation https://github.com/nabsabraham/focal-tversky-unet/blob/master/losses.py#L67, they calculate the FTL with `gamma=1/(4/3)=0.75`

Reply:
Yes indeed. I've modified it to `1 / gamma`.